.evergreen.yml

```diff
@@ -223,7 +223,6 @@ tasks:
     - func: get-project
     - func: sym-link
     - func: npm-install
-    - func: npm-lint
```

Comment:
This is replaced by eslint

Reply:
Can you delete the task entirely, and also the functions that would now be unused?

Reply:
Done.